api/src/main/java/net/jqwik/api/arbitraries/StreamableArbitrary.java

```diff
@@ -79,6 +79,6 @@ default StreamableArbitrary<T, U> ofSize(int size) {
 	 * @return new arbitrary instance
 	 */
 	@API(status = MAINTAINED, since = "1.7.3")
-	StreamableArbitrary<@Nullable T, U> uniqueElements(Function<@Nullable T, Object> by);
+	StreamableArbitrary<T, U> uniqueElements(Function<? super T, ?> by);
```

Comment:
The change to the second type parameter should then also entail a change in `@UniqueElements` annotation and the classes that process it.

Reply:
Why?

Reply:
`Class<? extends Function<?, ?>> by()` would allow specificly typed implementations. But it can also be done in a follow-up PR.